### PR TITLE
Update revoke-ome-encrypted-mail.md

### PR DESCRIPTION
**Pull request by samschan-ms**
Updated AAD roles that can access reovoke mail function. @KCCross to review

### CODE REVIEW

File: microsoft-365/compliance/revoke-ome-encrypted-mail.md [hunk 1]
```diff
@@ -127,13 +127,13 @@ Once you know the Message ID of the email you want to revoke, and you have verif
...
 
-1. Using a work or school account that has global administrator permissions in your organization, connect to the Microsoft Purview compliance portal.
+1. Using a work or school account that has global administrator or compliance administrator permissions in your organization, connect to the Microsoft Purview compliance portal.
 
 2. In the **Encryption report**, in the **Details** table for the message, choose **Revoke message**.
 
 To revoke an email by using Windows PowerShell, use the Set-OMEMessageRevocation cmdlet.
 
-1. Using a work or school account that has global administrator permissions in your organization, [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
+1. Using a work or school account that has global administrator, compliance administrator, Exchange administrator permissions, or other Exchange role groups with Information Right Management in your organization [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/revoke-ome-encrypted-mail.md]:
```suggestion
1. Using a work or school account that has global administrator, compliance administrator, or Exchange administrator permissions, or other Exchange role groups with Information Right Management in your organization, [connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Review by KCCross — CHANGES_REQUESTED** [review]
Will accept "or compliance administrator" "or Exchange administrator" however, we do not need to go into detail about IRM role here. That should be discussed one time (and one time only) in roles/perms content elsewhere. Since it's the IRM role that's granting the required permission for this and other procedures. Otherwise, we'd have to bring it up in every procedure. List the least-priv role/perm first, which I believe is compliance admin, then Exchange admin, then global administrator.

**Comment by KCCross** [comment]
After discussion, will merge.

**Pull request merged**